src/website/src/app/documentation/demos/accordion/accordion.demo.html

```diff
@@ -0,0 +1,38 @@
+<clr-doc-wrapper [ng]="ng" [ui]="ui" [title]="title" [newLayout]="newLayout">
+  <article>
+    <h5 class="component-summary" id="stepper-header">
+      A accordion allows generic content to be collapsed and allow users to expand to show more detail.
+    </h5>
+
+    <div id="design-guidelines">
+      <h3>Design Guidelines</h3>
+
+      <h3>Usage</h3>
+      <p>
+        Accordions are designed for use in the main content area. Use a accordion when you want to progressively
+        disclose data. This is useful when the set of information is large or you want to enable users to reveal more
+        complex or less frequently used data as necessary.
+      </p>
+    </div>
+
+    <div id="code-examples">
+      <h3 id="examples">Code &amp; Examples</h3>
+
+      <p>
+        Accordions by default only allow one panel to be open at a time. To enable multiple panels set
+        <code>clrAccordionMultiPanel="true"</code>
+        on the `clr-accordion` component.
```

Comment:
The back-ticks do not look well rendered.

<img width="230" alt="Screen Shot 2019-06-05 at 12 59 10 PM" src="https://user-images.githubusercontent.com/1798828/58947966-c2d11200-8791-11e9-8167-93d493f209b5.png">

Reply:
Ah markdown habits :)

Reply:
fixed